miniapps/common/dist_solver.cpp

```diff
@@ -264,7 +264,7 @@ void HeatDistanceSolver::ComputeScalarDistance(Coefficient &zero_level_set,
    if (vis_glvis)
    {
       char vishost[] = "localhost";
-      int  visport   = 19916;
+      int visport = 19916;
```

Comment:
Is this the intended change? The previous alignment is better IMHO

Reply:
This part of the change was scripted in python, I can adjust alignment manually